metrics_proto_test.go

```diff
@@ -69,13 +68,10 @@ func TestProtoResourceToMonitoringResource(t *testing.T) {
 		},
 	}
 
-	for i, tt := range tests {
+	for _, tt := range tests {
```

Comment:
Please keep this `i`, it is useful to see which test case it was.

Reply:
SG, done.